server/db/models/group.js

```diff
@@ -0,0 +1,52 @@
+const { Model, STRING, INTEGER, UUID, UUIDV4, BOOLEAN, TEXT } = require('sequelize');
+const db = require('../connection');
+
+class Group extends Model{}
+
+Group.init(
+    {
+        id: {
+            type: UUID,
+            defaultValue: UUIDV4,
+            unique: true,
+            primaryKey: true,
+        },
+        name: {
+            type: STRING,
+            unique: true,
+            allowNull: false,
+            validate: {
+                notEmpty: true
+            }
+        },
+        subject: {
+            type: STRING,
+            allowNull: false,
+            validate: {
+                notEmpty: true
+            }
+        },
+        topic: {
+            type: STRING,
+            allowNull: false,
+            validate: {
+                notEmpty: true
+            }
+        },
+        zipcode: {
+            type: INTEGER,
+            allowNull: false,
+            validate: {
+                notEmpty: true
+            }
+        },
+        description: {
+            type: TEXT,
+            allowNull: true
+        },
+        sequelize: db,
```

Comment:
The sequelize and model name keys need to be in their own object. See my user model.

Reply:
Ok, what do you think about Nick's suggestion as well? Do we want to have a duration for the event on the database?

Reply:
I think that sounds good.